extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-config.js

```diff
@@ -67,90 +77,52 @@ describes.realWin(
-          'productPrice': 10.0,
-          'productPriceCurrency': 'USD',
-          'productIcon':
-            '/examples/visual-tests/amp-story/img/shopping/icon.png',
-          'productImages': [
-            {
-              'url': 'https://source.unsplash.com/SavQfLRm4Do/500x500',
-              'alt': 'flowers',
-            },
-          ],
-          'aggregateRating': {
-            'ratingValue': 4.4,
-            'reviewCount': 89,
-            'reviewUrl': 'https://www.google.com',
-          },
-        },
       ],
     };
 
     const keyedDefaultInlineConfig = {
       'lamp': defaultInlineConfig.items[0],
       'art': defaultInlineConfig.items[1],
-      'chair': defaultInlineConfig.items[2],
-      'flowers': defaultInlineConfig.items[3],
     };
 
     beforeEach(async () => {
-      pageElement = env.win.document.createElement('amp-story-page');
-      pageElement.id = 'page1';
+      win = env.win;
+      storeService = getStoreService(win);
+      registerServiceBuilder(win, 'story-store', function () {
+        return storeService;
+      });
+      pageElement = <amp-story-page id="page1"></amp-story-page>;
+      win.document.body.appendChild(pageElement);
+
+      shoppingAttachment = win.document.createElement(
+        'amp-story-shopping-attachment'
+      );
+      shoppingAttachment.setAttribute('layout', 'nodisplay');
+      const story = win.document.createElement('amp-story');
+      win.document.body.appendChild(story);
+      story.appendChild(pageElement);
+      pageElement.appendChild(shoppingAttachment);
     });
 
     async function createAmpStoryShoppingConfig(
-      src = undefined,
+      src = null,
       config = defaultInlineConfig
     ) {
-      const shoppingAttachment = env.win.document.createElement(
-        'amp-story-shopping-attachment'
-      );
-      shoppingAttachment.setAttribute('layout', 'nodisplay');
       shoppingAttachment.setAttribute('src', src);
       shoppingAttachment.appendChild(
         <script type="application/json">{JSON.stringify(config)}</script>
       );
-      const story = env.win.document.createElement('amp-story');
-      env.win.document.body.appendChild(story);
-      story.appendChild(pageElement);
-
-      pageElement.appendChild(shoppingAttachment);
       return getShoppingConfig(shoppingAttachment);
     }
 
+    it('should build shopping config component', async () => {
+      expect(() => getShoppingConfig(shoppingAttachment)).to.not.throw();
+    });
+
     it('throws on no config', async () => {
-      expectAsyncConsoleError(async () => {
-        expect(() => {
-          const shoppingAttachment = <amp-story-shopping-attachment />;
-          pageElement.appendChild(shoppingAttachment);
-          return getShoppingConfig(shoppingAttachment);
-        }).to.throw(/<script> tag with type=\"application\/json\"​​​/);
-      });
+      expectAsyncConsoleError(
+        /<script> tag with type=\"application\/json\"​​​/
+      );
```

Comment:
General suggestion: To avoid the case where your tests pass due to some sort of mistake in how the test is written, it's generally a good idea to locally confirm that the tests begin to fail after changing the expectation to be incorrect.

So in the case of this PR, you would temporarily update this call to something slightly incorrect like `expectAsyncConsoleError(/<script> tag with story=\"application\/json\"/)`, also update the `expect()` calls in all the other tests with something slightly incorrect, and then run:
`amp unit --files=extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-config.js --headless --verbose`
Any test that still passes after doing this most likely needs to be updated.